### PR TITLE
adding chapter lead role and updating different things

### PR DESCRIPTION
**Pull request by jcmsalves**
- new page with open positions in the team
- moving interview sessions in one single document (roles now just have the structure)
- single page for the android/senior engineer role
- single page for the chapter lead role
- list of people updated

### CODE REVIEW

File: recruitment/positions/android_chapter_lead.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+Above all, we value one's ability to think independently and to engage.
+
+Additionally, we want to hire people who are reasonable, professional, 
+and fun to work with. After all, they are people we will spend a great 
+deal of time interacting with on a daily basis.
+
+We are continually improving how we work remotely as a company and aim 
+to be as flexible as possible by equally accepting applications from 
+candidates who want to work on-site at our office in London, or who 
+ant to work remotely.
```
Comment (athkalia) [inline recruitment/positions/android_chapter_lead.md]: `ant` -> `want`

File: recruitment/positions/android_chapter_lead.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+At Babylon we also believe in/champion diversity and welcome everyone 
+from everywhere. Currently in the Android team you'll find people from 
+Greece, Poland, UK, India, Spain, Portugal, France, Lithuania, Sudan, 
+Italy and Hungary, can you add to it?
```
Comment (athkalia) [inline recruitment/positions/android_chapter_lead.md]: maybe `and Hungary. Can you add to it?` then maybe an emoji if possible!

File: recruitment/positions/android_chapter_lead.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+The interview process for the Android Chapter Lead role generally 
+consists in the stages/sessions below. 1 and 2 always happen
+off-site and from 3-7 depending if you're based around London or not,
+some of the sessions might happen remotely as well but they usually
+happen in a half day on-site visit. 
+
+Check our [interview stages page](../interview_stages) for a brief 
+description of each of the stages/sessions.
+ 
+1. Initial phone call
```
Comment (athkalia) [inline recruitment/positions/android_chapter_lead.md]: you can put links for all of these I guess.

**Review by athkalia — CHANGES_REQUESTED** [review]

File: recruitment/positions/android_chapter_lead.md (hunk 2 above)
Comment (jcmsalves) [reply recruitment/positions/android_chapter_lead.md]: I added Hungary, it's there 👓

File: recruitment/positions/android_chapter_lead.md (hunk 3 above)
Comment (jcmsalves) [reply recruitment/positions/android_chapter_lead.md]: I would have to make the titles simpler in the other page, as they are right now it's really hard. Let me think about on how to make it better

Comment (athkalia) [reply recruitment/positions/android_chapter_lead.md]: cool.

File: recruitment/positions/android_chapter_lead.md (hunk 2 above)
Comment (jcmsalves) [reply recruitment/positions/android_chapter_lead.md]: replaced the list of countries with a line of flags

File: recruitment/positions/android_chapter_lead.md (hunk 3 above)
Comment (jcmsalves) [reply recruitment/positions/android_chapter_lead.md]: moved the duration and people involved out of the titles and added the links here and in the other role

**Review by athkalia — APPROVED** [review]

**Pull request merged**